pkg/authenticator/authenticator.go

```diff
@@ -113,7 +113,7 @@ func (auth *Authenticator) GenerateCSR(commonName string) ([]byte, error) {
 // successfully retrieved
 func (auth *Authenticator) Login() error {
 
-	log.Info(log.CAKC007I, auth.Config.Username)
+	log.Debug(log.CAKC007I, auth.Config.Username)
```

Comment:
this opens something up. we need to change `CAKC007I` to `CAKC007D` (or some other number) as the last letter indicates the log level. we have a few options here:
1. Change the suffix for every message that we move from Info to Debug
2. Remove the suffice altogether as the level is logged anyway in the same message.
3. Leave these messages with the `I` suffix and not have a `D` suffix at all (info and debug messages will have an `I` suffix).

The second option is the most correct one imo, although it is different from how we log in other projects (conjur, CSPFK). However, option 3 is backwards-compatible and doesn't require a doc change (also - customers may be monitoring these messages).

What do you think?

Reply:
Good point!
I agree that option 2 looks the most correct way to go, but it made me think: Does it mean that there could be 2 different messages, one for info and one for debug, but with the same ID?
It might create conflicts that will be hard to resolve.
And between 1 and 3, I prefer to go with the convention to decide, which is option 1.

Reply:
they won't have the same ID as one will be `CAKC001I` and one will be `CAKC001D`. we have the same thing with Error log messages. If we go for option 2 then we will need to restructure all the codes from the beginning and it will break compatibility. 

Option 1 is ok as well but it us not fully backwards-compatible. For example, a customer may monitor the log for `CAKC007I` but now it will change to `CAKC007D` and they will never catch it.

Option 3 is the safest one regarding backwards-compatibility. It has its flaws but I would go in that direction.

Reply:
I just read the [Semantic Versioning](http://semver.org/spec/v2.0.0.html) doc this project adheres to.
It explicitly says: [Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.](https://semver.org/spec/v2.0.0.html#spec-item-4)
Isn't it enough to allow the use of the 1st option?

Reply:
BTW, I also found this: https://semver.org/spec/v2.0.0.html#how-do-i-know-when-to-release-100.
@izgeri, why authn client wasn't released as 1.0.0 yet?

Reply:
i think we can go for option 1 but in that case then we can go with option 2. Which one do you prefer?

For both options I would add a CHANGELOG entry saying that the error codes were changed to enhance supportability.

Reply:
@abrahamko I don't know, but I've raised this with PM a couple of times recently. We will have to find a time to have a smallish focused effort on this project to get it to 1.0.0. cc @Tovli